pkg/sink/sink.go

```diff
@@ -108,17 +124,19 @@ func (r Sink) HandleEvent(response http.ResponseWriter, request *http.Request) {
 	log = log.With(zap.String("eventlistenerUID", elUID))
 	event, err := ioutil.ReadAll(request.Body)
 	if err != nil {
+		go cloudevent.SendCloudEvents(nil,
+			*el, eventID, cloudevent.TriggerProcessingFailedEventV1, r.TargetURI, r.CEClient, r.Logger)
```

Comment:
might be useful/cleaner to have a small helper in this file like `func (r Sink) SendCloudEvent(eventID, eventType string)`